rust/relay/tests/regression.proptest-regressions

```diff
@@ -0,0 +1,7 @@
+# Seeds for failure cases proptest has generated in the past. It is
+# automatically read and these particular cases re-run before any
+# novel cases are generated.
```

Comment:
That's cool

Reply:
Yesss. I would really like us to also write proptests for connlib, now that we are SANS-IO.